students/olszewski_bartosz/lesson_01_python_basics/school_desks.py

```diff
@@ -0,0 +1,12 @@
+print('Podaj liczebność klasy 1a')
+a = int(input())
+print('Podaj liczebność klasy 1b')
+b = int(input())
+print('Podaj liczebność klasy 1c')
+c = int(input())
+
+A_klasa = a//2+a%2
```

Comment:
Capital letters should be used only for constants e.g.
MY_NAME = "Kamil"

Please use lower case with underscores for variable name instead:
a_class = a // 2 + a % 2